storage/innobase/xtrabackup/test/t/xbcloud.sh

```diff
@@ -32,13 +56,14 @@ echo ${XBCLOUD_CREDENTIALS} | sed 's/ *--/\'$'\n/g' >> $topdir/xbcloud.cnf
 load_dbase_schema sakila
 load_dbase_data sakila
 
-mysql -e "CREATE database emptydatabas"
+mysql -e "ALTER TABLE payment COMPRESSION='lz4'" sakila
```

Comment:
why do we need this ALTER now?

Reply:
This changes exists in 8.0. so just back ported it. the test is successful